050_wpsage-plugin_L133.php

```
@@ -0,0 +1,389 @@
+            // 'media' => $this->get_media_info(),
+            // 'comments' => $this->get_comment_info(),
+            // 'taxonomies' => $this->get_taxonomy_info(),
+            'menus' => $this->get_menu_info(),
+            'widgets' => $this->get_widget_info(),
+            'favicon' => $this->get_favicon(),
+            'site_logo' => $this->get_site_logo(),
+        );
+        return new WP_REST_Response($response, 200);
+    }
+
+    public function get_health_check($request) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$request'. (undefined)

(UnusedFormalParameter)